azure-functions-sdk/src/commands/new_app.rs

```diff
@@ -117,6 +117,7 @@ impl<'a> NewApp<'a> {
                 json!({ "crate_version": env!("CARGO_PKG_VERSION") }),
             ),
             ("appsettings.json", "appsettings.json", json!({})),
+            ("local.settings.json", "local.settings.json", json!({})),
```

Comment:
Oh, we also need this declared in the `template!` macro in `src/commands.rs`.

Reply:
To test the SDK, you can run `cargo install -f --path azure-functions-sdk` from the repo root and then `cargo func new-app` to check the generated code.